### DISCUSSION TITLE
单纯 | /cn/2021/12/simplicity/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1499] [Comments]
如果吃饭只是吃饭，没有综艺
如果睡觉只是睡觉，没有手机
如果编程只是编程，没有阵营
如果购物只是购物，没有算法
如果看报只是看报，没有热搜
如果读书只是读书，没有勋章
如果写作只是写作，没有排行
如果游玩只是游玩，没有照片
如果唱歌只是唱歌，没有颜值
如果上学只是上学，没有报班
如果日记只是日记，没有票圈
如果天明只是天明，没有窗帘
如果月夜只是月夜，没有路灯
如果朋友只是朋友，没有人情
如果邻里 &hellip;

https://yihui.org/cn/2021/12/simplicity/

**Comment by ZaiyanHere** [comment]
如果此刻只是此刻，没有焦虑

**Reply by yihui** [reply]
根据定义，“在焉”就是活在当下，而要是真的活在当下，就不会过度焦虑（即：不要为明天忧虑，因为明天自有明天的忧虑）。

来，说说，你焦虑啥？让我听听，好治好我的焦虑。

**Reply by ZaiyanHere** [reply]
哈哈哈哈哈哈哈 yihui你完美地get到了我的名字啊！我暂时好像没什么焦虑的？等有了再来帮你治焦虑😂

**Reply by yihui** [reply]
那我就坐等你焦虑。

开个玩笑。不焦虑就好。我也没啥焦虑的。

我可不仅仅是完美阐释了你的网名。前几天我在琢磨你的本名是不是跟那句“我与我周旋久”有关，以及你爸爸给你取名字时是不是想到过这句话。

**Reply by ZaiyanHere** [reply]
我本名是爸爸的姓氏+妈妈的姓氏+一个我爸我妈都喜欢的字儿🙊 yihui真的明察秋毫 甚至我在领英看到过你哈哈哈！

**Reply by yihui** [reply]
本狗仔队队长被抓了个现行？看来功夫还没修炼到位。

**Reply by ZaiyanHere** [reply]
所以快通过我的好友申请～

**Reply by residualsun1** [reply]
> 根据定义，“在焉”就是活在当下，而要是真的活在当下，就不会过度焦虑（即：不要为明天忧虑，因为明天自有明天的忧虑）。

毕业论文初稿一字未写，但过几天就要交了，可是看到这段文字突然觉得没那么焦虑了，反正明天一样会忧虑，后天也会，大后天也会。如果现在开始就好好写，或许便不会感到十分焦虑吧。

**Reply by yihui** [reply]
来，码字达同学，奔袭起来吧。以你的码字力，几天时间足够了，区区一篇论文难不倒你的！（邢捕头：我看好你哟）

接下来几天我要是还看到你在到处闲晃，小心我宰了你……的 rbind.io 域名！

**Reply by yihui** [reply]
@residualsun1 恭喜你以迅雷不及掩耳盗铃之势几天就写完了初稿！我就说嘛，区区一篇论文对你肯定是小菜一碟，而且毕业论文这种东西说实话我觉得一点也不重要，甚至我可以武断地讲，你写的博客对全宇宙的贡献和价值肯定远大于你的毕业论文。另外也恭喜你考研的佳绩！想来上一次我进民大校园还是 2002 年，你都还没出生吧，哈哈。

**Reply by residualsun1** [reply]
> 恭喜你以迅雷不及掩耳盗铃之势几天就写完了初稿！我就说嘛，区区一篇论文对你肯定是小菜一碟

那天受到你留言后我还真老老实实把自己关起来写初稿了，接下来将是反反复复的修改（悲，好折磨）。

> 甚至我可以武断地讲，你写的博客对全宇宙的贡献和价值肯定远大于你的毕业论文。另外也恭喜你考研的佳绩！想来上一次我进民大校园还是 2002 年，你都还没出生吧，哈哈。

谢谢你，益辉！不管是否武断，都非常非常鼓励我 :D！！感谢你的祝贺(11◡‿◡)，能收到你的祝贺非常开心！！2002年再往后挪一年我才出生，转眼 22 年，我也要去北京读书了^ ^！

**Reply by yihui** [reply]
哦耶，到时候你一出校门，不管左拐右拐，都会踩到很多我在你负一岁时留下的脚印以及自行车轮子印！

国家图书馆就在你们隔壁，这下你可以尽情地在书籍的海洋中迷失自我了。加油！

**Reply by residualsun1** [reply]
现在民大很多专业的研究生前两年都要被“流放”到丰台新校区இ௰இ，第三年才能回去了。感觉海淀好多好地方，我复试那天从学校东门一出来，隔壁是国家图书馆，另一边好像就是北大口腔医院，而且清华北大人大也都在那儿，以后我多找机会过去！
